paper/paper.md

```diff
@@ -0,0 +1,126 @@
+
+## Main Features
+pyOBIS python package improvess accessibility of data available through OBIS
+and helps reduce efforts in manipulating and visualizing Darwin Core Data.
+Some of the key features of pyOBIS are:
+* **Easy handling of OBIS data**
+
+  Users can easily fetch data without handling the API directly.
+  The comprehensive documentation and built-in funtions provides support to both beginners and experienced researchers in handling Darwin Core Data.
+  Response is always returned as a custom object with pre-defined methods to export to a `pandas` DataFrame,
+  generate live API URL to plugin to any additional software, and
+  build an OBIS Mapper URL for direct one-click visualization on the OBIS Mapper portal.
+
+* **Smart download, processing and export of data**
+
+  pyOBIS provides an interactive progress bar while fetching large occurrence records.
+  It also provides an estimated size of the request and the expected time to taken for the download.
+  pyOBIS un-nests entangled occurrence data, and increases readibility for beginner users.
+  It provides easy export of data to Pandas DataFrame,
+  so that researchers can export it to any format like `csv`, `excel`, `JSON` making data handling and compatibility
+  with other software super-easy.
+
+* **Richer support with sister packages**
+
+  pyOBIS when coupled with sister packages e.g. `pyDwcViz` can be utilized to perform many important computations easily.
+  With one-line function and plug-and-play use,
+  users can generate biodiversity indices such as `ES50` and `Shannon's Index`,
+  get environment statistics from occurrence records queried for specified geo-spatial region of interest,
+  taxa, or other paramters,
+  generate interactive distribution plots with taxanomic heirarchy easily,
+  and many other possible use cases.
+
+# Figures
+![Absolute Depth for Lepidochelys kempii over time.\label{fig:time-series-turtle}](https://github.com/ayushanand18/pyobis/assets/36472216/b6e66f31-7bbd-49c9-8186-3ab1a58e57c0)
+
+pyOBIS can be used to do super-useful time series analysis for instance, absolute depth of Sea Turtle species, Lepidochelys kempii between 1990-2011 as shown in figure \autoref{fig:time-series-turtle}. From this analysis, the following observations can be made:
+* The average depth has increased over the years, this means the species is looking for cooler waters to escape the heating waters. (This can be observed from the magenta-colored line which depicts the 5-year rolling average.)
+* The species has witnessed a slight compression, i.e., minimal and maximal depth have come closer. For a brief period, it compressed significantly (around 2006) this might be due to data constraints or maybe some seasonal current. After that it has regained a lot but still the average difference in minimal and maximal depth is lower than early 2000s.
+* However, necessary precautions to avoid sampling bias must be taken into consideration.
+
+# Conclusion
```

Comment:
I have added a conclusion to the paper. Should we mention something else too? Like some open-ended research questions which pyOBIS can help solve. 

cc @ocefpaf @7yl4r

Reply:
We could pull some ideas from the specific research questions that could be addressed with OBIS data [in marinebon/obis-use-cases/issues](https://github.com/marinebon/obis-use-cases/issues), but I like the conclusion as-is.